operator/README.md

```diff
@@ -1,2 +1,92 @@
-# pmem-csi-operator
-Operator for PMEM-CSI
+# PMEM-CSI-Operator
+
+Kubernetes operator for [PMEM-CSI](https://github.com/intel/pmem-csi) driver deployment. Using this operator one can easily deploy and manage PMEM-CSI driver in a Kubernetes cluster.
+
+The PMEM-CSI operator is based on the CoreOS [operator-sdk](https://github.com/operator-framework/operator-sdk) tools and APIs.
+
+## PMEM-CSI Deployment custom resource definition(CRD)
+
+Current PMEM-CSI Deployment object supports below API:
+
+### Deployment
+
+|Field | type | Description |
+|---|---|---|
+| apiVersion | string  | `pmem-csi.intel.com/v1alpha1`|
+| kind | string | `Deployment`|
+| metaData | [ObjectMeta](https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status) | Standard objects metadata. |
+| spec | [DeploymentSpec](#deployment-spec) | Specification of the desired behavior of the deployment |
+
+### DeployemtSpec
+
+|Field | type | Description | Default Value |
+|---|---|---|---|
+| image | ImageInfo | PMEM-CSI driver container image information | `intel/pmem-csi-driver:canary` |
+| logLevel | integer | PMEM-CSI driver logging level | 3 |
```

Comment:
Currently, we have no description of logging levels in our documentation.

Reply:
good point, we could add note in README (or better DEVELOPMENT ?) about our style, which seems to be off-by-one from guidelines at https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
i.e. we use (Info levels) 3 for production level and 4,5 for extended/development.
I recall the history was, when we cleaned our logging levels, we had most of statements at level 3 already so we wanted to keep those and we used 4 and 5 as more verbose levels.
 We do not use info levels 0,1,2 currently at all.
At some point, do we wish to change all levels so that we conform better to convention in abovementioned page?